build.py

```diff
@@ -76,7 +76,7 @@
         "2024.0.0",  # ORT OpenVINO
         "2024.0.0",  # Standalone OpenVINO
         "3.2.6",  # DCGM version
-        "0.5.0.post1",  # vLLM version
+        "v0.5.3.post1",  # vLLM version
```

Comment:
Should the `v` be included?

Reply:
good catch, I'll remove it for consistency with previous specs. CI passed though with `pip install vllm==v0.5.3.post1` though

Reply:
adjusted

Reply:
Oh nice, I didn't know it would work both ways, just thought it was a typo. No preference either way if it works.